jest.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  testRegex: "((\\.|/*.)(spec))\\.js?$",
+  moduleNameMapper: {
+    "^.+\\.(css|less|scss)$": "babel-jest",
+    "\\.svg": "<rootDir>/svgMock.js"
+  },
+  setupFilesAfterEnv: [
+    "<rootDir>/src/enzyme.js"
+  ],
+  testPathIgnorePatterns: [
+    "<rootDir>/cypress"
+  ]
```

Comment:
You need collectCoverageFrom config to collect coverage from all your project (under src directory), otherwise files without tests are not counted.